apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -0,0 +1,156 @@
+import { Round } from "../../(utils)/rounds";
+import ClientSideBodyRow from "./ClientSideBodyRow";
+
+export default async function PoolListTable({ roundId }: { roundId: string }) {
+  return (
+    <div className="overflow-hidden rounded-lg w-full text-white">
+      <Table color="blue" variant="solid" shade="dark">
+        <Table.HeaderRow>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>TVL</Table.HeaderCell>
+          <Table.HeaderCell> Votes % </Table.HeaderCell>
+          <Table.HeaderCell>APR</Table.HeaderCell>
+        </Table.HeaderRow>
+        <Table.Body>
+          {votingGauges.slice(0, 10).map((gauge) => (
+            <TableRow
+              poolId={gauge.pool.id}
+              network={gauge.network}
+              roundId={roundId}
+              key={gauge.address}
+            />
+          ))}
+          <Table.BodyRow>
+            <Table.BodyCell colSpan={4}>
+              <button
+                className="w-full flex content-center justify-center gap-x-3 bg-blue4 py-2 text-sm disabled:opacity-50 disabled:cursor-not-allowed hover:cursor-pointer hover:bg-blue5 hover:text-white"
+                disabled={true}
+              >
+                Load More <ChevronDownIcon />
+              </button>
+            </Table.BodyCell>
+          </Table.BodyRow>
+        </Table.Body>
+      </Table>
+    </div>
+  );
+}
+
+export async function PoolVotes({
+  roundId,
+  pool,
+}: {
+  roundId?: string;
+  pool: Pool;
+}) {
+  if (!roundId) return null;
+
+  const votingShare = await getPoolRelativeWeight(
+    pool.id,
+    Round.getRoundByNumber(roundId).endDate.getTime() / 1000,
+  );
+
+  return <>{(votingShare * 100).toFixed(2)}%</>;
+}
+
+export async function PoolTVL({ pool }: { pool: Pool }) {
+  const TotalValueLocked = await BalancerAPI.getPoolTotalLiquidityUSD(
+    pool.gauge?.network || 1,
+    pool.id,
+  );
+  return <>{formatNumber(TotalValueLocked)}</>;
+}
+
+export async function PoolAPR({
+  roundId,
+  pool,
+}: {
+  roundId?: string;
+  pool: Pool;
+}) {
+  // TODO: aggregate historical pool APR when roundId is not provided
+  if (!roundId) return null;
+
+  const round = Round.getRoundByNumber(roundId);
+  const [balPriceUSD, tvl, votingShare] = await Promise.all([
+    getBALPriceByRound(round),
+    // TODO: must select the correct network
+    BalancerAPI.getPoolTotalLiquidityUSD(pool.gauge?.network || 1, pool.id),
+    getPoolRelativeWeight(pool.id, round.endDate.getTime() / 1000),
+  ]);
+
+  const apr = calculateRoundAPR(round, votingShare, tvl, balPriceUSD) * 100;
+
+  return <>{apr?.toFixed?.(2)}%</>;
+}
+
+function TableRow({
+  network,
+  poolId,
+  roundId,
+}: {
+  poolId: string;
+  network: number;
+  roundId: string;
+}) {
+  const pool = new Pool(poolId);
+  return (
+    <ClientSideBodyRow
+      poolId={pool.id}
+      poolNetwork={network}
+      roundId={roundId}
+      key={pool.id}
+    >
```

Comment:
I assume this was needed because <a> tags within <tr> is not valid HTML. IMO we should just refactor the table/th/tr/td out and use tailwind utilities: `.table, .table-row, .table-cell`.

Reply:
logged [here](https://linear.app/bleu-llc/issue/BAL-650/refactor-table-to-use-tailwind-classes-and-use-link-instead-of)